ROMFS/px4fmu_common/init.d/3100_tbs_caipirinha

```diff
@@ -54,7 +54,11 @@ then
 	param set PWM_MIN 900
 	param set PWM_MAX 2100
 
-	param set SYS_COMPANION 157600
+	# TELEM2 config
+	param set MAV_1_CONFIG 2
+	param set MAV_1_RATE 1000
+	param set MAV_1_MODE 3
+	param set SER_TEL2_BAUD 57600
```

Comment:
In my opinion this should be dropped. There's nothing about a TBS caipirinha that requires a specific telemetry 2 configuration. The airframe can easily be kept generic.

Reply:
Agreed, dropped.